node/pkg/processor/vaa.go

```diff
@@ -26,16 +24,10 @@ func (v *VAA) HandleQuorum(sigs []*vaa.Signature, hash string, p *Processor) {
 		Payload:          v.Payload,
 		ConsistencyLevel: v.ConsistencyLevel,
 	}
-	vaaBytes, err := signed.Marshal()
-	if err != nil {
-		panic(err)
-	}
 
 	// Store signed VAA in database.
 	p.logger.Info("signed VAA with quorum",
```

Comment:
agree with ditching the whole bytes in logs, as it's hefty and unknown size, but may want to consider still logging the signatures.

Reply:
After an offline discussion, we decided not to log the sigs, because the message grows from 280 bytes to 2800 bytes.